### PR TITLE
adds browser tracing / profiling, reenables server profiling

### PR DESCRIPTION
**Pull request by stolinski**

### CODE REVIEW

**Comment by stolinski** [comment]
Going to merge to get the browser profiling stuff in. Will make a new branch for node profiling.

File: src/hooks.client.ts [hunk 1]
```diff
@@ -18,7 +18,12 @@ Sentry.init({
 	replaysOnErrorSampleRate: 1.0,
 
 	// If you don't want to use Session Replay, just remove the line below:
-	integrations: [new Replay()],
+	integrations: [
+		new Sentry.BrowserTracing(),
```
Comment (Lms24) [inline src/hooks.client.ts:22]: The Sveltekit SDK already adds `BrowserTracing` (not `BrowserProfiling` though) to the integrations. Did this not work correctly for you?

Comment (stolinski) [reply src/hooks.client.ts:22]: Gotcha, so if using Sentry.init from `'@sentry/sveltekit'` we don't need to add the integration? TBH I just saw that I wanted to add BrowserTracing, went to the docs, saw the instructions and added.

Comment (Lms24) [reply src/hooks.client.ts:22]: > so if using Sentry.init from '@sentry/sveltekit' we don't need to add the integration?

correct you only need to add `BrowserProfiling`.

Ahh I just saw that the [browser profiling docs](https://docs.sentry.io/platforms/javascript/guides/sveltekit/profiling/#configure) include `BrowserTracing`. Thanks for letting me know, I'll fix this in the docs.

Comment (stolinski) [reply src/hooks.client.ts:22]: Glad to help. Thanks for the info.

**Pull request merged**